package.json

```diff
@@ -87,6 +87,7 @@
   "dependencies": {
     "rc-slider": "^8.6.2",
     "react": "^16.5.0",
+    "react-spring": "^5.7.2",
```

Comment:
Palette will have an animation section fairly shortly so it was expected that this dependency would be added in time.